wps-cloudformation-template.yaml

```diff
@@ -277,6 +292,15 @@ Resources:
           -
             Name: "RETRY_INTERVAL_MS"
             Value: !Ref workerRetryIntervalMs
+
+            Name: "GEONETWORK_CATALOGUE_URL"
+            Value: !Ref geonetworkCatalogueURL
+
+            Name: "GEONETWORK_LAYER_SEARCH_FIELD"
+            Value: !Ref geonetworkLayerSearchField
+
```

Comment:
PR looks good. Just a small issue with the merge. These parameters should be in the list.

```yaml
          -
            Name: "RETRY_INTERVAL_MS"
            Value: !Ref workerRetryIntervalMs
          -
            Name: "GEONETWORK_CATALOGUE_URL"
            Value: !Ref geonetworkCatalogueURL
          -
            Name: "GEONETWORK_LAYER_SEARCH_FIELD"
            Value: !Ref geonetworkLayerSearchField
          -
            Name: "PROVENANCE_TEMPLATE_S3_KEY"
            Value: !Join [ "", [ !Ref s3configKey,  !FindInMap [ConfigurationFileMap, "Filename", "provenanceTemplateGridded"]] ]
```